pkg/apis/certificates/validation/validation.go

```diff
@@ -572,3 +579,263 @@ func validateTrustBundle(path *field.Path, in string) field.ErrorList {
 
 	return allErrors
 }
+
+// We don't care what you call your pod certificate requests.
+func validatePodCertificateRequestName(name string, prefix bool) []string {
+	return nil
+}
+
+// ValidatePCRCreate checks validation for create.
+//
+// When creating, all spec fields are validated, and all status fields must be
+// empty.
+func ValidatePCRCreate(req *certificates.PodCertificateRequest) field.ErrorList {
+	var allErrors field.ErrorList
+
+	metaErrors := apivalidation.ValidateObjectMeta(&req.ObjectMeta, true, validatePodCertificateRequestName, field.NewPath("metadata"))
+	allErrors = append(allErrors, metaErrors...)
+
+	signerNameErrors := apivalidation.ValidateSignerName(field.NewPath("spec", "signerName"), req.Spec.SignerName)
+	allErrors = append(allErrors, signerNameErrors...)
+
+	// PodName, PodUID, ServiceAccountName, ServiceAccountUID, NodeName, and
+	// NodeUID have already been checked to be real values by the
+	// noderestriction admission plugin.
+
+	if req.Spec.MaxExpirationSeconds < 3600 {
+		allErrors = append(allErrors, field.Invalid(field.NewPath("spec", "maxExpirationSeconds"), req.Spec.MaxExpirationSeconds, "must be >= 3600 (1 hour)"))
+	}
+
+	// Check key and proof of possession.
+	allErrors = append(allErrors, validatePCRSpecCrypto(req)...)
+
+	// All status fields must be empty when created.
```

Comment:
strategy should make this unnecessary with `PrepareForCreate`

Reply:
Done.